src/components/common/layout/base.njk

```diff
@@ -3,6 +3,20 @@
 {% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
 
 {% block head %}
+    <script>
```

Comment:
this won't work as the CSP blocks any inline scripts from running for security reasons